monkey/infection_monkey/readme.md

```diff
@@ -33,16 +33,22 @@ The monkey is composed of three separate parts.
 		build_windows.bat 
 		output is placed under dist\monkey.exe
 
---- Linux ---
-
-Tested on Ubuntu 16.04 and 17.04.
+## Linux
 
-1.	Install dependencies by running:
-		sudo apt-get update
-		sudo apt-get install python-pip python-dev libffi-dev upx libssl-dev libc++1
+Tested on Ubuntu 16.04.
+0. On older distributions of Ubuntu (16.04) you'll need to download python3.7 via ppa:
```

Comment:
Is this still the current version tested?

Reply:
yes